skype/skype-ps/skype/grant-CsTeamsCallHoldPolicy.md

```diff
@@ -0,0 +1,227 @@
+online version: https://docs.microsoft.com/powershell/module/skype/grant-CsTeamsCallHoldPolicy
+Module Name: MicrosoftTeams
+online version:
+schema: 2.0.0
+title: grant-CsTeamsCallHoldPolicy
+manager: abnair
+author: joelhmarquez
+ms.author: jomarque
+ms.reviewer:
+---
+
+# grant-CsTeamsCallHoldPolicy
+
+## SYNOPSIS
+Assigns a per-user Teams call hold policy to one or more users.
+The Teams call hold policy is used to customize the call hold experience for Teams clients.
+
+## SYNTAX
+
+### Identity (Default)
+```
+grant-CsTeamsCallHoldPolicy [[-Identity] <UserIdParameter>] [-PolicyName] <String> [-Tenant <System.Guid>]
+ [-DomainController <Fqdn>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### GrantToTenant
+```
+grant-CsTeamsCallHoldPolicy [-PolicyName] <String> [-Tenant <System.Guid>] [-DomainController <Fqdn>]
+ [-PassThru] [-Global] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Teams call hold policies are used to customize the call hold experience for teams clients.
+When Microsoft Teams users participate in calls, they have the ability to hold a call and have the other entity in the call listen to an audio file during the duration of the hold.
+Assinging a teams call hold policy to a user sets an audio file to be played during the duration of the hold.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Grant-CsTeamsCallHoldPolicy -Identity 'KenMyer@contoso.com' -PolicyName 'ContosoPartnerTeamsCallHoldPolicy'
+```
+
+The command shown in Example 1 assigns the per-user Teams call hold policy, ContosoPartnerTeamsCallHoldPolicy, to the user with the user principal name (UPN) "KenMyer@contoso.com".
+
+### Example 2
+```powershell
+PS C:\> Grant-CsTeamsCallHoldPolicy -Identity 'Ken Myer' -PolicyName 'ContosoPartnerTeamsCallHoldPolicy'
+```
+
+The command shown in Example 2 assigns the per-user Teams call hold policy, ContosoPartnerTeamsCallHoldPolicy, to the user with the display name "Ken Myer".
+
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsCallHoldPolicy -Identity 'Ken Myer' -PolicyName $null
+```
+
+In Example 3, any per-user Teams call hold policy previously assigned to the user "Ken Myer" is revoked.
+As a result, the user will be managed by the global Teams compliance recording policy.
+
+## PARAMETERS
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DomainController
+This parameter is reserved for internal Microsoft use.
+
+```yaml
+Type: Fqdn
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Global
+When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant.

```

Comment:
Seems that including `-Global` makes the selected audio the new Global

![image](https://user-images.githubusercontent.com/36666927/119501151-d5013d80-bd25-11eb-9e14-56355ecdb8e5.png)
![image](https://user-images.githubusercontent.com/36666927/119501195-de8aa580-bd25-11eb-815f-8621917fdd26.png)

Can you kindly state this in the description? Thank you

Reply:
Added another example showing this